ext/GeometryOpsLibGEOSExt/simple_overrides.jl

```diff
@@ -0,0 +1,60 @@
+#=
+
+# Simple overrides
+
+This file contains simple overrides for GEOS, essentially only those
+functions which have direct counterparts in LibGEOS and only
+require conversion before calling.
+
+=#
+# ## Polygon set operations
+# ### Difference
+function GO.difference(::GEOS, geom_a, geom_b; target=nothing)
+    return LG.difference(GI.convert(LG, geom_a), GI.convert(LG, geom_b))
```

Comment:
LG has these converts build in if you want to simplify this code a bit

Reply:
I've had issues before but will test if this works.  Possibly that was on simplify, which isn't implemented in LibGEOS

Reply:
Yeah, maybe I missed a few methods. Might be good to list those and we can fix them for everyone else too.

Reply:
Yeah, I have hit a few of those. I believe `makeValid` and `isValid` only work with `LG` polygons as well.